Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -264,8 +264,14 @@ def check_for_latest_xquartz
 
         <<-EOS.undent
           Your XQuartz (#{installed_version}) is outdated.
-          Please install XQuartz #{latest_version} (or delete it):
-            https://xquartz.macosforge.org
+          Please install XQuartz #{latest_version} (or delete the current version).
```

Comment:
That was there before - I was just clarifying. I believe the reason it says that is because the diagnostic will go away in either case: both upgrading and removing will satisfy `brew doctor`.